projects/go-lib/src/lib/components/go-datepicker/calendar-views.scss

```diff
@@ -59,15 +69,52 @@
   }
 
   &.go-calendar__cell--selected {
-    background-color: $ui-color-neutral-hover;
-    color: $theme-dark-text-color;
+    font-weight: normal;
+    background-color: $ui-color-primary;
+    color: $theme-dark-color;
   }
 }
 
-.go-calendar__body__border {
-  border-top: 1px solid black;
+.go-calendar__cell--date {
+  width: 1.375rem;
 }
 
 .go-calendar__table {
   width: 100%;
 }
+
+.go-calendar__body--padding {
+  padding-top: .5rem;
+}
+
+.go-calendar__row {
+  .go-calendar__date:first-child, .go-calendar__body--header:first-child {
+    padding-left: 1rem;
+  }
+
+  .go-calendar__date:last-child, .go-calendar__body--header:last-child {
+    padding-right: 1rem;
+  }
+}
+
+.go-calendar__row:last-child {
+  .go-calendar__date:last-child {
+    padding-right: 0;
+  }
+
+  .go-calendar__date {
+    padding-bottom: .625rem;
+  }
+
+  .go-calendar__date--tall {
+    padding-bottom: 1.5rem;
+  }
+}
+
+:host ::ng-deep .go-icon--small {
```

Comment:
I don't think we need to do this. Shouldn't we be able to apply this class directly to the `go-icon` element?

Reply:
Yes you are correct here